lib/devise-security.rb

```diff
@@ -83,6 +83,14 @@ module Devise
   # Whether to allow passwords that are equal (case insensitive) to the email
   mattr_accessor :allow_passwords_equal_to_email
   @@allow_passwords_equal_to_email = false
+
+  # Custom redirect route for successful password expiration update action
+  mattr_accessor :password_expired_redirect_location
+  @@password_expired_redirect_location = nil
```

Comment:
We can just set the default here instead of in the controllers

```suggestion
  @@password_expired_redirect_location = :root
```